phoenix-hive/src/main/java/org/apache/phoenix/hive/PhoenixMetaHook.java

```diff
@@ -94,8 +89,12 @@ private String createTableStatement(Table table) throws MetaException {
         String phoenixRowKeys = tableParameterMap.get(PhoenixStorageHandlerConstants
                 .PHOENIX_ROWKEYS);
         StringBuilder realRowKeys = new StringBuilder();
-        List<String> phoenixRowKeyList = Lists.newArrayList(Splitter.on
-                (PhoenixStorageHandlerConstants.COMMA).trimResults().split(phoenixRowKeys));
+        List<String> phoenixRowKeyList = new ArrayList<>(
```

Comment:
You must not use Java 8 features. Phoenix 4.x targets Java 7, and the connectors repo has to work with it.